Include/internal/pycore_global_objects.h

```diff
@@ -34,6 +34,24 @@ extern "C" {
     }
 
 
+/* bytes objects */
+
+#define _PyBytes_EMPTY_INIT \
+    { \
+        _PyVarObject_IMMORTAL_INIT(&PyBytes_Type, 0), \
+        .ob_shash = -1, \
+        .ob_sval[0] = '\0', \
+    }
+#define _PyBytes_CHAR_INIT(CH) \
```

Comment:
This is a declaration, not initialization code. Maybe rename to `_PyBytes_CHAR`?

Reply:
I was sticking to the `_INIT` convention used elsewhere.